omero/developers/Web/CreateApp.txt

```diff
@@ -9,11 +9,11 @@ database mapping, since all data comes from the OMERO server and is
 saved back there. You will notice that the models.py files in each app
 are empty.
 
-.. note:: Since OMERO 5.0, the web framework uses Django 1.6 instead of Django
-    1.3. One important change is the syntax of the
-    :djangodoc:`url template tag <ref/templates/builtins/#std:templatetag-url>`,
-    which now requires quotes, and will need to be updated for OMERO 4.4 web
-    apps moving to OMERO 5.0.
+.. note:: Since OMERO 5.2, the web framework is no loger shipped with Django
```

Comment:
Do you need to repeat this message? You're already referencing `:doc:Deployment` below which contains this warning.